### PR TITLE
[READY] New KPP copy and images

### PR DESCRIPTION
**Pull request by RonaldDijkstra**
- [x] add new text by Hans
- [x] smaller H1 heading
- [x] remove hero cta
- [x] remove commented out css
- [x] add kwaliteitspaspoort and kwaliteitsportfolio images to NL page
- [ ] ~~add images to the bottom list~~

### CODE REVIEW

**Comment by RonaldDijkstra** [comment]
This is ready for review. 

What is changed is some new text, which should not be reviewed at this point, but just deployed. 

I didn't add the images for the list in the bottom, I need more time to gather these. I will do them in another PR.

File: source/stylesheets/_kpp.scss [hunk 1]
```diff
@@ -10,19 +10,42 @@
...
 
-      // @include media($large-up) {
-      //   font-size: 4em;
-      // }
+      @include media($large-down) {
+        font-size: 2em;
+      }
+    }
+
+    p {
```
Comment (fatboypunk) [inline source/stylesheets/_kpp.scss]: @sn3p what to do with these?

File: source/stylesheets/_layout.scss [hunk 2]
```diff
@@ -267,3 +267,8 @@ Contact info
   font-style: italic;
   text-align: center;
 }
+
+figcaption {
+  font-style: italic;
+  text-align: center;
+}
```
Comment (sn3p) [inline source/stylesheets/_layout.scss]: Lets combine and move these declarations to [_typography.scss](https://github.com/DefactoSoftware/website/blob/master/source/stylesheets/base/_typography.scss):

```scss
figcaption,
.caption {
  font-style: italic;
  text-align: center;
}
```

File: source/stylesheets/_kpp.scss (hunk 1 above)
Comment (sn3p) [inline source/stylesheets/_kpp.scss]: Ignore for now but lets config hound to accept 4 or 5 in the future

**Pull request merged**